### DISCUSSION TITLE
API : What do the 2nd numbers refer to for connected inputs?

### DISCUSSION THREAD

**Discussion by caniyabanci76** [comfyanonymous/ComfyUI#1506] [Q&A] [answered]
in the API output we have something like the following for the VAEDecode node :

```
  "8": {
    "inputs": {
      "samples": [
        "3",
        0
      ],
      "vae": [
        "4",
        2
      ]
    },
    "class_type": "VAEDecode"
  },
```

for `samples` and `vae`, I understand that the first number strings "3", "4" refer to the node IDs that they connected to, but what is the secondary number? (in this case 0 and 2) ?

**Comment by ltdrdata** [comment, marked as answer]
2nd number is slot number
